source/includes/_organizations.md

```diff
@@ -77,3 +77,187 @@ Get a list of active job postings for a company.
 | Parameter | Description                                                                                                                 | Example                  |
 | --------- | --------------------------------------------------------------------------------------------------------------------------- | ------------------------ |
 | id        | The id of the organization (i.e. NOT account id). You can obtain this with a company's domain by using the enrich endpoint. | 54fca1087369647fcXXXXXXX |
+
+
+
+## Organization search 
+
+> Sample request:
+
+```shell
+curl -X POST -H "Content-Type: application/json" -H "Cache-Control: no-cache" -d '{
+  "api_key": "YOUR API KEY HERE",
+  "page": 1,
+  "per_page": 10,
+  "organization_num_employees_ranges": ["1,100", "1,1000"],
+  "organization_locations": ["United States"],
+  "q_organization_keyword_tags": ["sales strategy", "lead"],
+  "q_organization_name": "Apollo.io"
+}' "https://api.apollo.io/api/v1/mixed_companies/search"
+``` 
+
+```python
+import requests
+
+url = "https://api.apollo.io/api/v1/mixed_companies/search"
+
+data = {
+    "api_key": "YOUR API KEY HERE",
+    "page":1,
+    "per_page":10,  
+    "organization_num_employees_ranges":["1,100", "1,1000"],
+    "organization_locations":["United States"],
+    "q_organization_keyword_tags":["sales strategy", "lead"],
+    "q_organization_name":"Apollo.io"
+}
+
+headers = {
+    'Cache-Control': 'no-cache',
+    'Content-Type': 'application/json'
+}
+
+response = requests.request("POST", url, headers=headers, json=data)
+
+print(response.text)
+```
+
+> Sample response:
+
+```json 
+{
+    "breadcrumbs": [
+        {
+            "label": "Company Name",
+            "signal_field_name": "q_organization_name",
+            "value": "Apollo.io",
+            "display_name": "Apollo.io"
+        },
+        {
+            "label": "# Employees",
+            "signal_field_name": "organization_num_employees_ranges",
+            "value": "1,100",
+            "display_name": "1-100"
+        },
+        {
```

Comment:
This is repeating, value has been changed. is this intentional ?

Reply:
These are auto-constructed from the breadcrumbs. Does that answer?

Reply:
Okay, just want to understand, whether this has been added by mistake

Reply:
Can we resolve this @PavanSankavar